assignments/this.js

```diff
@@ -1,26 +1,54 @@
 /* The for principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
 *
-* 1. 
-* 2. 
-* 3. 
-* 4. 
+* 1. Window Binding - this is the global object of whatever is bound to "THIS". this is the most overall binding. If you use "use strict" within a function, then you can never override it... as you shouldn't!
+
+* 2. Implicit Binding - seen most of the time in people's code. This is when a function is called, it looks to the left of the dot to see what object it should be looking to when it's invoked.
+
+* 3. New Binding - This one is when you use a constructor. A blank blueprint of what an obect should be. it uses the "THIS" keyword. When you create a new object, you base it off the constructor by using this "New" keyword binding
+
+* 4. Explicit binding - this is useful for when you want to run the same function on numerous objects and would like to only write it out once. You can use .call() to invoke it on the object you wish. It is built outside and seperately from the inside of objects which helps with the "DRY" concept.
 *
 * write out a code example of each explanation above
 */
 
-// Principle 1
+// Principle 1 - Window Binding
+function sayAge(age) {
+  console.log(this);
+  return age;
+}
```

Comment:
In your Principle 1 example, which line is your example of a Window Binding:
`console.log(this);`
OR
`return age;`?

If you don't think that it's the `console.log()` let me know so that I can explain it!